src/components/clientSettings/clientSettings.tsx

```diff
@@ -0,0 +1,190 @@
+import { useTranslation } from 'react-i18next';
+import { Controller, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import Select from 'react-select';
+import {
+	checkList,
+	Data,
+	ICountry,
+	initial,
+	initialCountry,
+	selection,
+	ValidationSchema,
+} from 'components/clientSettings/data';
+import {
+	CheckLabel,
+	Column,
+	Container,
+	SaveButton,
+	CancelButton,
+	MainTitle,
+	P,
+	Span,
+	Title,
+	Div,
+	Input,
+	TextArea,
+	MinColumn,
+	MaxColumn,
+	ButtonBlock,
+	Button,
+	Img,
+} from 'components/clientSettings/clentSettings.styles';
+import countryList from 'react-select-country-list';
+import { FormEvent, useMemo, useState } from 'react';
+import editIcon from 'image/icon-pencil.png';
+import ChangePassword from 'pages/setting-page-client/change-password/ChangePassword';
+
+const settings = 'settings';
+const password = 'password';
```

Comment:
I think it should b in general constant file

Reply:
fixed